app/controllers/projects_controller.rb

```diff
@@ -96,7 +96,7 @@ def initialze_projects
 
   def filter_projects_list_by_language
     @language = params[:project][:languages]
```

Comment:
Hey @diraulo, thanks for the PR review, I am using the `@language` instance variable

Reply:
good question @tansaku, I guess that the projects would still be sorted by language cause it would be in the url, but it would not be selected in the dropdown. 

Another use case would be if someone shared a deep link with a friend, then  they would see the search params in the url, but not in the dropdown.

I guess it would be good to leave it then.